thirdparty/patches/avx2ki-2.0.0-operation.patch

```diff
@@ -0,0 +1,19 @@
+--- /usr/local/ksl/include/operatoroverload.h	2023-09-14 18:39:40.000000000 +0800
++++ /usr/local/ksl/operatoroverload.h	2023-12-19 23:11:28.546116818 +0800
```

Comment:
is this the right location to patch?

Reply:
the real location to patch is determined by the parameter passed to the patch command. The location here is jsut the temporary location when making this patch.